crates/trans_token/Cargo.toml

```
@@ -20,8 +20,10 @@ migrations = [
 
 [dependencies]
 namada_core = { path = "../core" }
+namada_events = { path = "../events", default-features = false }
```

```suggestion
namada_events = { path = "../events" }
```
tiny (there seem to be no default features)

---

I added this because eventually I'd like to move the event log out of `namada_sdk` to this crate